src/main/scala/edu/ie3/powerFactory2psdm/config/validate/conversion/ConversionModeValidators.scala

```diff
@@ -0,0 +1,173 @@
+/*
+ * © 2021. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.powerFactory2psdm.config.validate.conversion
+
+import edu.ie3.powerFactory2psdm.config.model.PvConversionConfig.{
+  PvFixedFeedIn,
+  PvModelConversionMode,
+  PvModelGeneration
+}
+import edu.ie3.powerFactory2psdm.config.model.WecConversionConfig.{
+  WecFixedFeedIn,
+  WecModelConversionMode,
+  WecModelGeneration
+}
+import edu.ie3.powerFactory2psdm.config.validate.ConfigValidator.{
+  validateCpCharacteristic,
+  validateParameterSamplingMethod,
+  validateQCharacteristic
+}
+import edu.ie3.powerFactory2psdm.exception.io.ConversionConfigException
+
+import scala.util.{Failure, Success}
+
+object ConversionModeValidators {
+
+  object PvConversionModeValidator
+      extends ConversionModeValidator[PvModelConversionMode] {
+
+    def apply(conversionMode: PvModelConversionMode): Unit = {
```

Comment:
you may call this method `validate(...)` instead of `apply(...)` as by convention apply is the factory method and not a regular method ....